.travis.yml

```diff
@@ -4,10 +4,12 @@ node_js:
 cache:
   directories:
     - node_modules
+before_install:
+  - npm install -g npm@latest
 before_script:
   - chmod +x scripts/docs
 script:
-  - npm run cover
+  - CI=true npm run cover
```

Comment:
https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables
The suits themselves are running slow. It ends up showing the timing for each suit to indicate that a particular suit took longer than expected to run. That indicates that the tests are taking a long time and not that it is hanging waiting for user input.
It runs pretty quick locally for me as well. I am think it might be because I have a powerful machine and travis is running in a VM with little processing power. idk.